Objects/typeobject.c

```diff
@@ -4845,12 +4895,18 @@ add_members(PyTypeObject *type, PyMemberDef *memb)
     PyObject *dict = type->tp_dict;
 
     for (; memb->name != NULL; memb++) {
-        PyObject *descr;
-        if (PyDict_GetItemString(dict, memb->name))
-            continue;
-        descr = PyDescr_NewMember(type, memb);
+        PyObject *descr = PyDescr_NewMember(type, memb);
```

Comment:
It seems a little strange to break the short-circuit like this (here and in a few other places in this PR), but I see your intent: use `GetItem` instead of `GetItemString`.  As long as the performance (and semantics) impact is minimal that should be fine.